bike-tutorial/python/bike-tutorial.py

```diff
@@ -14,17 +13,19 @@ def create_schema(client):
         client : a WOQLClient() connection
 
     """
-    station = WOQLQuery().doctype("Station").label("Bike Station")
```

Comment:
Yes, both of them work (in the newest version - v0.0.10) and here we show that we can break it down into multiple variables. As a Python user, what do you think is better?

I think it doesn't work on yours becuase I have just patch it in the new version so it does not need to reassign it back to the variable (e.g. journey) Sorry I was updating the tutorial too soon. If you pip install the latest python client it will work.

Reply:
Hmm my personal preference is to just stick to the fluent approach (as in the politics example) - casting each type of entity as an instance can become tedious and repetitive for a real case (it kind of looks ugly in my opinion to just keep typing the same instance name again an again). Also, I'd think that people would already know they can create individual instances that way if they really wanted to, so again, I'd just avoid it altogether and reduce the number of repetitive instances.

Reply:
Ah ok, good to know. Both should work so we will keep your suggetion as the tutorial. Thanks!